docs/source/tools.rst

```diff
@@ -0,0 +1,73 @@
+
+Tools Guide
+=================
```

Comment:
Title underline === should be same length as the title, I got warnings about this in my own environment when I was working on the 91 docs

Reply:
I did not get any warnings, but yes needs to be changed

Reply:
Make underline === same length as Tools Guide.